### PR TITLE
Update headline feature & manual for GCE changes

### PR DESCRIPTION
**Pull request by kgaikwad**
Screenshots need to be updated for GCE changes so marking it as WIP.

### CODE REVIEW

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ This section will be updated prior to the next release.
 Global parameters now support the same types as smart variables and smart class parameters.
 As a part of this change searching by parameter value is no longer allowed. ([#4127](https://projects.theforeman.org/issues/4127))
 
+#### Google Compute Resource
+
+Google Compute Resource has been fixed now as due to changes in google compute APIs, previously it was broken. The cloud-init integration also being added to google compute resource.
```
Comment (tbrisker) [inline _includes/manuals/nightly/1.2_release_notes.md]:
```suggestion
Google Compute Resource has been fixed now. Due to changes in google compute APIs, it was previously broken. cloud-init integration has also been added to google compute resource. There are few edge cases which are still need to be handled, but basic provisioning should be functional.
```

File: _includes/manuals/nightly/5.2.4_cr_gce.md [hunk 2]
```diff
@@ -1,40 +1,48 @@
...
 * The certificate must be stored in a location the foreman user account has permission to read.
-* If your server enforces SELinux ensure the context is suitable or relabel it using `restorecon -vv /usr/share/foreman/gce.p12`
-* Specify the location *on the foreman server* as the certificate path value e.g /usr/share/foreman/gce.p12
-* Ensure images are associated under the *Images* tab on the compute resource.
-* Add a provisioning template of type *finish* which will be executed over SSH on the new image.
-* Ensure the finish template is associated to the OS (on the *Associations* tab) and is set as the default for the operating system too.
+* If your server enforces SELinux ensure the context is suitable or relabel it using `restorecon -vv /usr/share/foreman/gce.json`
+* Specify the location *on the foreman server* as the certificate path value e.g /usr/share/foreman/gce.json
+* Ensure images are associated under the *Images* tab on the compute resource
+	* Ensure the correct username except *root* is set for Foreman to SSH into the image (if using SSH provisioning). Note that - For security reasons, Google do not provide the ability to ssh in directly as root.
```
Comment (tbrisker) [inline _includes/manuals/nightly/5.2.4_cr_gce.md:6]: not sure if it's intentional but some of the dots seem to be indented?

**Review by tbrisker — COMMENTED** [review]
just a small rewording please, otherwise looks good.

File: _includes/manuals/nightly/5.2.4_cr_gce.md (hunk 2 above)
Comment (kgaikwad) [reply _includes/manuals/nightly/5.2.4_cr_gce.md:6]: Yes, to keep description format consistent with other CR, I have added it intentionally.
These sub-points are related to image so placed it under *Images* Tab.

**Comment by kgaikwad** [comment]
Updated PR with the suggested changes.

**Review by tbrisker — APPROVED** [review]
Thanks @kgaikwad !

**Pull request merged**